tls/s2n_config.h

```diff
@@ -107,6 +107,8 @@ struct s2n_config {
 
     unsigned custom_blinding_set : 1;
 
+    unsigned disable_legacy_tickets : 1;
```

Comment:
we should avoid `disable_*` for flags since it becomes hard to reason about. I would just call it `legacy_tickets`.